### PR TITLE
Add the dhcp-gateway and dns-reverse entries for installer-options

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

**Comment by jyejare** [comment]
ACK

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -285,6 +285,15 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         'capsule-puppetca': 'true',
         'capsule-register-in-foreman': 'true',
     }
+
+    if os.environ.get('GATEWAY'):
```
Comment (elyezer) [inline automation_tools/__init__.py]:
```python
if 'GATEWAY' in os.environ:
```

**Comment by elyezer** [comment]
ACK, pending little nitpick.

**Comment by elyezer** [comment]
APT

**Pull request merged**